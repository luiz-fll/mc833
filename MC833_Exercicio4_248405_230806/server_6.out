remoto: 127.0.0.1:6006
[SERVIDOR] Escutando em 127.0.0.1:6006
127.0.0.1:53072
GET / HTTP/1.0
Host: teste

127.0.0.1:53078
GET / HTTP/1.0
Host: teste

127.0.0.1:53088
GET / HTTP/1.0
Host: teste

127.0.0.1:53094
GET / HTTP/1.0
Host: teste

127.0.0.1:53104
GET / HTTP/1.0
Host: teste

127.0.0.1:53120
GET / HTTP/1.0
Host: teste

127.0.0.1:53126
GET / HTTP/1.0
Host: teste

127.0.0.1:53132
GET / HTTP/1.0
Host: teste

127.0.0.1:53146
GET / HTTP/1.0
Host: teste

