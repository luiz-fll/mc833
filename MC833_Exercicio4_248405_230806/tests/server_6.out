remoto: 127.0.0.1:50006
[SERVIDOR] Escutando em 127.0.0.1:50006
127.0.0.1:41396
GET / HTTP/1.0
Host: teste

127.0.0.1:41406
GET / HTTP/1.0
Host: teste

127.0.0.1:41412
GET / HTTP/1.0
Host: teste

127.0.0.1:41420
GET / HTTP/1.0
Host: teste

127.0.0.1:41430
GET / HTTP/1.0
Host: teste

127.0.0.1:41436
GET / HTTP/1.0
Host: teste

127.0.0.1:41438
GET / HTTP/1.0
Host: teste

127.0.0.1:41444
GET / HTTP/1.0
Host: teste

127.0.0.1:41448
GET / HTTP/1.0
Host: teste

127.0.0.1:41452
GET / HTTP/1.0
Host: teste

